raster_download/local-deps/copy_pg_bins.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
```

Comment:
Could you please add a short code comment what this script does? I have an idea due to the file name, but for better maintainability it would be good to have at least a one-liner.